internal/commands/config/config.go

```diff
@@ -44,10 +50,15 @@ func NewCommand() *cobra.Command {
 }
 
 func runConfig(cmd *cobra.Command, args []string) error {
+	gitlabRunnerDriver := map[string]string{
+		driverName:    tart.TartCommandName,
```

Comment:
Shouldn't this be just:

```suggestion
		driverName:    "gitlab-tart-executor",
```

?

Reply:
the driver name is supposed to describe the ecosystem/technology the executor is using, not the executor itself.

the [log message](https://gitlab.com/gitlab-org/gitlab-runner/-/blob/d9df8321d93f92a3ba63d40076ae98b902b3848f/executors/custom/custom.go#L224) would read as follows for the tart executor:

> Using Custom executor with driver tart 1.22.0-deadbeefcafe